src/redisraft.c

```diff
@@ -649,6 +649,10 @@ static void handleRedisCommandAppend(RedisRaftCtx *rr,
         RaftRedisCommand *cmd = cmds->commands[i];
         size_t cmdlen;
         const char *cmdstr = RedisModule_StringPtrLen(cmd->argv[0], &cmdlen);
+        /* skip multi */
+        if (i == 0 && cmdlen == 5 && !strncasecmp(cmdstr, "MULTI", 5)) {
+            continue;
+        }
```

Comment:
Why do we skip MULTI? I remember we discussed this and decide we shouldn't skip it, no? Because dry-run can also reject MULTI.

Reply:
in regular redis multi isn't enqueued, hence not skipping it would cause a problem as a change in multi acl shouldn't impact anything now.

Reply:
You are right again and I'm still not happy :) 

Btw we should have one flag variable in the entry to encode these. `multi`, `asking` etc can be a bit of a single variable. If we have it, our entry size will be smaller and we can avoid this check here and also on apply time. We can do it later ofcourse as part of another PR.